doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -478,8 +478,26 @@ Multi-image build
 
 To build and program a multi-image HEX file, follow the instructions in :ref:`programming_cmd` for the application core sample.
 
-To program the multi-image HEX file, you must use west.
-Programming multi-image builds for different cores is not yet supported in nrfjprog.
+To program the multi-image HEX file, you can use west or nrfjprog.
+
+.. tabs::
+
+   .. group-tab:: west
+
+      Enter the following command to program multi-image builds for different cores::
+
+        west flash
+
+   .. group-tab:: nrfjprog
+
+      Enter the following commands program multi-image builds for different cores::
+
+        nrfjprog -f NRF53 --program zephyr/merged_domains.hex --recover --verify
+
+      .. note::
+         The minimum supported version of nrfjprog for multi-image builds for different cores is 10.21.0.
+
+         Additionally, the ``--verify`` command confirms that the write succeeded.
```

Comment:
```suggestion
         Additionally, the ``--verify`` command confirms that the write operation has succeeded.
```
may be we could say right operation in this case?
We could move this statement after line no: 496 (out of the note admonitions) since this statement talks about the command, so it suits there.

Reply:
Hm, I think it is referring to the "writing" operation.
And sure, this could be moved out side of the admonition - there is a similar case on line 531-540 though. It is much longer. With this in mind, should the case on line 531 also be moved? Or should both stay as admonitions for consistency?